src/client/src/components/pages/participants/viewParticipants/index.jsx

```diff
@@ -53,9 +54,46 @@ export default class ViewParticpants extends Component {
     }
   };
 
-  onChangeName = (event) => {
-    const nameSearch = event.target.value;
-    this.setState({ nameSearch }, () => this.nameSearcher());
+  surnameSearcher = async () => {
+    const { surenameSearch } = this.state;
+    const data = await axios('/api/v2/participants/search/surname', {
+      method: 'POST',
+      data: {
+        participantName: surenameSearch,
+      },
+    });
+    const finalData = data.data.searchResult;
+    if (finalData) {
+      const array = [['BB_No.', 'fullname', 'Date Of Birth', 'borough', 'Email', 'Action']];
+      finalData.map(row => array.push([
+        row.id,
+        `${row.forename} ${row.surename}`,
+        row.date_of_birth.split('T')[0],
+        row.borough,
+        row.email,
+          <>
+            <Link to={`/participant/details/${row.id}`}>
+              <i className="fas fa-info-circle" />
+            </Link>
+            <i className="fas fa-trash-alt" onClick={() => this.onDelete(row.id)} />
+          </>,
+      ]));
+      this.setState({ rows: array });
+    } else {
+      const array = [];
+      const msg = data.data.message;
+      this.setState({ message: msg, rows: array });
+    }
```

Comment:
@mohannadhanafi Hey! If this project were going to continue, a good next step to this code would be to refactor the common parts of forename and surname search into a single function, since they do a lot of the same work.